pkg/client/core/service_identitymanager.go

```diff
@@ -35,6 +35,10 @@ func (s *identityManagerService) DeleteIdentityPool(ctx context.Context, provide
 
 	pool := s.state.GetIdentityPool()
 
```

Comment:
Not really related, but if you try to run delete again and the user-pool is missing, it will continue instead of crashing ...